app/models/user.rb

```diff
@@ -25,7 +25,7 @@ class User < ActiveRecord::Base
   has_many :collection_roles, through: :identity_group
   has_many :user_seen_subjects
 
-  has_many :subject_queues, dependent: :destroy
```

Comment:
Yeah, when deleting on the console (maybe not us but private instances) i'd expect that stuff to get cleaned up.